app/serializers/serialize_submission.rb

```diff
@@ -4,12 +4,20 @@ class SerializeSubmission
   initialize_with :submission
 
   def call
+    return unless submission
+
     {
       uuid: submission.uuid,
       tests_status: submission.tests_status,
       links: {
-        cancel: Exercism::Routes.api_submission_cancellations_url(submission, auth_token: user.auth_tokens.first.to_s)
-      }
+        cancel: Exercism::Routes.api_submission_cancellations_url(submission, auth_token: user.auth_tokens.first.to_s),
+        submit: Exercism::Routes.api_solution_iterations_url(
```

Comment:
I decided to add the link to submit the iteration here.

Reply:
That's a nice decision, yes.

Reply:
Yes I like that!